demos/benchmarks/README.md

```diff
@@ -0,0 +1,86 @@
+# PowerSync Benchmark app
```

Comment:
I don't have a strong opinion, but I'm thinking this might fit better in a tools/ directory, similar to what we have for the diagnostics app

Reply:
There's quite a bit of tooling in this repo specific to the demos folder, so it would take some refactoring to do that